locations/spiders/cineworld_gb.py

```diff
@@ -11,7 +11,7 @@ class CineworldGBSpider(Spider):
     name = "cineworld_gb"
     item_attributes = {"brand": "Cineworld", "brand_wikidata": "Q5120901"}
     start_urls = ["https://www.cineworld.co.uk/"]
-    requires_proxy = True
+    requires_proxy = "GB"
```

Comment:
The theory is that it would default to the spider location, from the name cineworld_**gb**. So we don't need to be explicit in most cases.